getting-started/deploying-to-openshift.adoc

```diff
@@ -0,0 +1,235 @@
+----
+oc login
+----
+
+. **Add the Docker extension to the Quarkus project:**
++
+[source,shell]
+----
+mvn quarkus:add-extension -Dextensions='quarkus-container-image-docker'
+----
+
+. **Update the `src/main/resources/application.properties` file with the following properties to configure the image:**
++
+[source,properties]
+----
+# Container Image Builder
+quarkus.container-image.builder=docker
+quarkus.container-image.registry=<image_registry>
+quarkus.container-image.group=<registry_group_or_user>
+quarkus.container-image.name=<image_name>
+quarkus.container-image.tag=<image_tag>
+quarkus.docker.buildx.platform=linux/amd64
+----
+
+[NOTE]
+====
+* `<image_registry>`: The URL of the container image registry where the Docker image will be stored. Example: `docker.io`.
+* `<registry_group_or_user>`: The group or user in the container image registry. Example: `myuser`.
+* `<image_name>`: The name of the Docker image to be built. Example: `myapp`.
+* `<image_tag>`: The tag of the Docker image. Example: `v1.0.0`.
+
+Replace these tokens with appropriate values based on the environment setup.
+====
+
+. **Add the `Dockerfile.jvm` file to `src/main/docker`:**
++
+[source,Dockerfile]
+----
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jre-ubi-minimal
+
+# Switch to root
+USER 0
+
+# Copy the compiled application files from the local `target` directory to the `/app/` directory inside the container.
+COPY target /app/
+
+# Support Arbitrary User IDs
+# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
+RUN chgrp -R 0 /app && \
+  chmod -R g+rwX /app
+
+# Switch back to a non-root user
+USER 1001
+
+WORKDIR /app
+
+EXPOSE 8080
+
+CMD ["java", "-jar", "quarkus-app/quarkus-run.jar"]
+----
+
+. **Build and push the Docker image:**
++
+[source,shell]
+----
+mvn install -Dquarkus.container-image.build=true -Dquarkus.container-image.push=true
+----
+
+. **Manually create a deployment and expose it on OpenShift:**
++
+[source,shell]
+----
+oc new-app --name=<deployment_name> <image_registry>/<registry_group_or_user>/<image_name>:<image_tag>
+oc create route edge --service=<deployment_name>
+----
+
+[NOTE]
+====
+* `<deployment_name>`: Is the name of the OpenShift deployment that will be created. Example: `myapp-deployment`.
+====
+
+== Conclusion
+
+Choose the appropriate deployment method based on the development workflow and environment requirements. For further customization or troubleshooting, refer to the official Quarkus guides.
+
+- https://quarkus.io/guides/container-image
+- https://quarkus.io/guides/deploying-to-openshift
+
+___
+
+[NOTE]
+====
+The first option will use an UBI OpenJDK base image from RedHat (usually `registry.access.redhat.com/ubi9/openjdk-17`) while the second and third options are based on the IBM Semeru image (`icr.io/appcafe/ibm-semeru-runtimes:open-17-jre-ubi-minimal`), which can be customized by editing the `Dockerfile.jvm` file in `src/main/docker`.
```

Comment:
Red Hat. While it may repeat, I think it is better to have this info at the top of each of the three methods.